### PR TITLE
fig3 extending and result titles

### PR DESCRIPTION
**Pull request by Farnazmdi**

### CODE REVIEW

**Comment by Farnazmdi** [comment]
@shak360 I made figure 3 so that the lineage is extending with depth and breadth.

File: lineage/figures/figureS53.py [hunk 1]
```diff
@@ -25,7 +26,7 @@ def makeFigure():
 
     # Get list of axis objects
     ax, f = getSetup((10, 10), (3, 3))
-    lin_params = {"pi": pi, "T": T, "E": E2, "desired_num_cells": min_desired_num_cells}
+    lin_params = {"pi": pi, "T": T, "E": E2}
```
Comment (shak360) [inline lineage/figures/figureS53.py:29]: I would delete this line.
```suggestion
```

File: lineage/figures/figureS53.py [hunk 2]
```diff
@@ -25,7 +26,7 @@ def makeFigure():
...
     number_of_columns = 25
     figureMaker3(ax, *accuracy(lin_params, number_of_columns))
```
Comment (shak360) [inline lineage/figures/figureS53.py:31]: I would delete the argument `lin_params` argument.
```suggestion
    figureMaker3(ax, *accuracy(number_of_columns))
```

File: lineage/figures/figureS53.py [hunk 3]
```diff
@@ -44,19 +45,19 @@ def accuracy(lin_params, number_of_columns):
     """
```
Comment (shak360) [inline lineage/figures/figureS53.py:45]: I would delete the argument here as well (in the function definition)

File: lineage/figures/figureS53.py [hunk 4]
```diff
@@ -44,19 +45,19 @@ def accuracy(lin_params, number_of_columns):
...
-    for num in num_lineages:
+    for indx, num in enumerate(num_lineages):
         population = []
-
         for _ in range(num):
 
             good2go = False
             while not good2go:
-                tmp_lineage = LineageTree.init_from_parameters(**lin_params)
+                tmp_lineage = LineageTree.init_from_parameters(pi, T, E2, desired_num_cells=num_cells[indx])
```
[the remark this answers is not in the document]
Comment (Farnazmdi) [reply lineage/figures/figureS53.py:60]: No, just S53.

**Comment by Farnazmdi** [comment]
@shak360 Thanks for the comments, I applied them. Will merge now.

**Pull request merged**